src/bproxy.c

```diff
@@ -278,17 +277,17 @@ void proxy_connect_cb(uv_connect_t *req, int status)
   {
     conn_close(conn);
     // TODO: write meaningful responses
-    return;
```

Comment:
is it right to remove that `return;` and continue reading from socket even if connection has closed?

Reply:
Thanks for this comment, It's a mistake.